lib/rucio/client/uploadclient.py

```diff
@@ -234,22 +239,20 @@ def upload(self, items, summary_file_path=None, traces_copy_out=None):
                 trace['protocol'] = cur_scheme
                 trace['transferStart'] = time.time()
                 try:
-                    state = rsemgr.upload(rse_settings=rse_settings,
-                                          lfns=lfn,
-                                          domain=domain,
-                                          source_dir=file['dirname'],
-                                          force_scheme=cur_scheme,
-                                          force_pfn=pfn,
-                                          transfer_timeout=file.get('transfer_timeout'),
-                                          delete_existing=delete_existing,
-                                          sign_service=sign_service,
-                                          auth_token=self.auth_token,
-                                          logger=logger)
-                    success = state['success']
-                    file['upload_result'] = state
+                    self._upload_item(rse_settings=rse_settings,
+                                      lfn=lfn,
+                                      source_dir=file['dirname'],
+                                      force_scheme=cur_scheme,
+                                      force_pfn=pfn,
+                                      transfer_timeout=file.get('transfer_timeout'),
+                                      delete_existing=delete_existing,
+                                      sign_service=sign_service)
```

Comment:
I think you are missing the auth_token (OIDC) here. That functionality was introduced to the rse manager by @gumond rather recently, but I think this OR would essentially take the passthrough out of the upload workflow again. Or am I missing something?

Reply:
the auth. is resoved when creating a protocol instance, not at the upload itself. But I'll rather cehck that.